src/easy.rs

```diff
@@ -2864,6 +2951,11 @@ impl Easy {
         }
     }
 
+    fn getopt_double(&mut self, opt: curl_sys::CURLINFO) -> Result<c_double, Error> {
+        self.getopt_ptr(opt)
+            .map(|p| unsafe {transmute::<_, c_double>(p)})
```

Comment:
This looks to be failing on 32-bit platforms, perhaps this could just call the FFI fn directly?